SU-HSDO/suhumsci
# Release: FourKitchens Sprint 10

* [shs-108: School name overlapping menu on Mobile on some phones](https://fourkitchens.atlassian.net/browse/SHS-108) 
* [shs-111: Table view without header causes a broken admin menu and link issues](https://fourkitchens.atlassian.net/browse/SHS-111) (JS Error) 

# READY FOR REVIEW

## Summary
- Prevent Table Pattern JS from throw an error if table row headings don't exist
- Fix header layout that is specifically affecting smaller screen iphones. 

## Need Review By (Date)
10/04

## Urgency
low

## Steps to Test
Table fix:
1. Create a flexible page with a view that has table headers. I chose 'Default Courses' as I was able to replicate the bug.
2. View the page and make sure the table headers are present, there's no console errors related to the table and JS still works (navigation dropdown for example).

iPhone fix:
- Test with iphone11 or iphoneSE type of screens or use Xcode iOS simulator or BrowserStack
Verify the header is consistent across devices (14 vs SE):

<img width="906" alt="Screen Shot 2022-09-30 at 4 23 57 PM" src="https://user-images.githubusercontent.com/369018/193359029-fa56a13e-7ce2-4302-9022-b3c653d0c42e.png">

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
